pkgs/desktops/plasma-5/default.nix

```diff
@@ -164,6 +162,8 @@ let
         parachute = callPackage ./3rdparty/kwin/scripts/parachute.nix { };
       };
 
+    } // lib.optionalAttrs (config.allowAliases or true) {
+      ksysguard = throw "ksysguard has been replaced with plasma-systemmonitor";
```

Comment:
I'm confused about this: won't this cause an error to be thrown when `allowAliases = true`?

Reply:
Yes, exactly like how other such warnings work -- compare to pkgs.ytop, as a randomly chosen example.

When aliases are enabled, we produce errors, when they're disabled, the attribute just doesn't exist.

Reply:
That the name "allow" means "throw an error" is worse than useless, but that's not your fault. :frowning: